src/styles/getMuiTheme.js

```diff
@@ -4,6 +4,21 @@ import ColorManipulator from '../utils/color-manipulator';
 import autoPrefix from './auto-prefix';
 import lightBaseTheme from './baseThemes/lightBaseTheme';
 import zIndex from './zIndex';
+import {autoprefixer, rtl} from './transformers';
+import compose from 'lodash.flowright';
+import warning from 'warning';
+
+const CALLED_ONCE = Symbol('prepareStyles/calledOnce');
```

Comment:
I think we should avoid polyfills for the time being. es6 has to become mainstream before we can use it libraries. I use every feature i can get my hands on in my application. but libraries are different :grin: